scio-google-cloud-platform/src/it/scala/com/spotify/scio/bigquery/BigQueryIT.scala

```diff
@@ -37,14 +38,24 @@ object BigQueryIT {
   class WordCount
 }
 
-class BigQueryIT extends AnyFlatSpec with Matchers {
+class BigQueryIT extends AnyFlatSpec with Matchers with BeforeAndAfterEach with BeforeAndAfterAll {
   import BigQueryIT._
 
   // =======================================================================
   // Integration test with mock data
   // =======================================================================
 
-  "MockBigQuery" should "support mock data" in {
+  val StagingDatasetPrefix = "scio_bigquery_staging_custom"
+
+  override def beforeEach() {
+    sys.props.subtractOne(BigQuerySysProps.StagingDatasetPrefix.flag)
```

Comment:
`SystemProperties` APIs `addOne` and `subtractOne` are only available for scala 2.13
Scala 2.12 only supports `+=` and `-=` operators